src/Resources/Interfaces/ErrorInterface.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace MyParcelCom\ApiSdk\Resources\Interfaces;
+
+use JsonSerializable;
+
+interface ErrorInterface extends JsonSerializable
+{
+    /**
+     * @param string $id
+     * @return $this
+     */
+    public function setId($id);
+
+    /**
+     * @return string
+     */
+    public function getId();
+
+    /**
+     * @param array $links
+     * @return $this
+     */
+    public function setLinks(array $links);
+
+    /**
+     * @return array
+     */
+    public function getLinks();
+
+    /**
+     * @param string $status
+     * @return $this
+     */
+    public function setStatus($status);
+
+    /**
+     * @return string
+     */
+    public function getStatus();
+
+    /**
+     * @param string $code
+     * @return $this
+     */
+    public function setCode($code);
+
+    /**
+     * @return string
+     */
+    public function getCode();
+
+    /**
+     * @param string $title
+     * @return $this
+     */
+    public function setTitle($title);
+
+    /**
+     * @return string
+     */
+    public function getTitle();
+
+    /**
+     * @param string $detail
+     * @return $this
+     */
+    public function setDetail($detail);
```

Comment:
According to the spec, this should be `details`

Reply:
Accoding to the jsonapi specification it should be `detail` though. I think this should be fixed in the api spec rather than the sdk.

Reply:
Yup, I created a Jira issue for the spec.